app/controllers/application_controller.rb

```diff
@@ -28,7 +28,7 @@ class MissingParameterError < RuntimeError; end
     @message = exception.message
     layout = request.xhr? ? false : 'application'
     logger.error exception.backtrace.join("\n") unless EXCEPTIONS_TO_IGNORE.include? exception
-    render template: 'error', formats: [:html], layout: layout, status: 400
+    render template: 'error', formats: [:html], layout:, status: 400
```

Comment:
I don't like that style at all, I think it's pretty obscure. I would rather set in `.rubocop.yml`

```
Style/HashSyntax:
  EnforcedShorthandSyntax: either
```

Reply:
Fair, I will revert these